posts/app.py

```diff
@@ -0,0 +1,66 @@
+"""
+posts/app.py
+Authors: mukobi
+Main flask app for posts
+    - Serve stored media
+    - Upload new posts to database
+
+
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import os
+
+from flask import Flask, jsonify, request
+
+app = Flask(__name__)
+
+
+@app.route('/v1/add', methods=['POST'])
+def upload_new_post():
+    """
+    Make a new post upload to the server.
```

Comment:
First line of docstring goes on the same line as `"""` (here and below)

`"""If you have a single-line comment, it should look like this."""`

Reply:
Done https://github.com/knative-portability/large-events/pull/3/commits/9948e4b2a5a2bdecfa61b9b156c37882fa2320bd